CRM/Contribute/BAO/Contribution.php

```diff
@@ -3592,6 +3604,28 @@ public static function updateFinancialAccounts(&$params, $context = NULL, $skipT
         }
       }
     }
+    if ($context == 'changePaymentInstrument') {
```

Comment:
@pradpnayak all this is doing is adding in a linkage back to the financial_item without it the display on the contribution tab does not show all financial_trxns